From 1315c88bb3a77a555ec11317ff20aa01ed731afa Mon Sep 17 00:00:00 2001
From: Frank Hunleth <fhunleth@troodon-software.com>
Date: Mon, 2 Jun 2014 18:11:44 -0400
Subject: [PATCH 5/8] libarchive: add host-libarchive

This enables host compilation of libarchive so that host tools using it
can be built with buildroot.
---
 package/libarchive/libarchive.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/package/libarchive/libarchive.mk b/package/libarchive/libarchive.mk
index 206de3f..f45335e 100644
--- a/package/libarchive/libarchive.mk
+++ b/package/libarchive/libarchive.mk
@@ -93,3 +93,4 @@ LIBARCHIVE_CONF_OPTS += --without-lzma
 endif
 
 $(eval $(autotools-package))
+$(eval $(host-autotools-package))
-- 
2.5.0

